app/src/localization/fr-FR.json

```diff
@@ -211,7 +211,7 @@
   "email-slash-username": "E-m-a-i-l-/-U-s-e-r-n-a-m-e",
   "password-confirmation": "P-a-s-s-w-o-r-d- -C-o-n-f-i-r-m-a-t-i-o-n",
   "search-results": "S-e-a-r-c-h- -R-e-s-u-l-t-s",
-  "search-results-for": "S-e-a-r-c-h- -r-e-s-u-l-t-s- -f-o-r-:- -'{searchKeywords}'",
+  "search-results-for": "S-e-a-r-c-h- -r-e-s-u-l-t-s- -f-o-r-:- -'-{-s-e-a-r-c-h-K-e-y-w-o-r-d-s-}-'",
```

Comment:
@dusanradovanovic It breaks a variable in the text, need to update translate script.